### DISCUSSION TITLE
Add Keycloak SAML to the documentation

### DISCUSSION THREAD

**Discussion by VincentSC** [getredash/redash#5963] [Q&A]
Hi, I just got finished with https://www.linkedin.com/pulse/redash-login-via-keycloak-saml-vincent-hindriksen/ to explain how to get Redash working with Keycloak.

Would it be a good idea to add this to the [documentation](https://redash.io/help/user-guide/users/authentication-options)? If yes, I can rewrite it into the style of that page.

**Comment by arikfr** [comment]
This is not related to the issue, but I assume that the issue author or followers might have SAML enabled for their deployment and should be aware of the following Security Advisory: https://github.com/getredash/redash/discussions/5961. This affects *all* Redash versions and should be patched immediately.

**Comment by arikfr** [comment]
And to answer your question: sure! This will be much appreciated. Thanks.
